packs/equipment-effects/effect-bombers-eye-elixir.json

```diff
@@ -22,31 +22,46 @@
         },
         "rules": [
             {
-                "actorFlag": true,
                 "adjustName": false,
                 "choices": [
                     {
-                        "label": "PF2E.SpecificRule.Equipment.BombersEyeElixir.Lesser",
                         "predicate": [
                             {
                                 "not": "parent:origin:item:slug:bombers-eye-elixir-greater"
                             }
                         ],
-                        "value": 1
```

Comment:
Would we need slugs as values set here?

Reply:
The roll option automatically has the slug of the item the uuid points to